### PR TITLE
[Chore] Update docs for programming exercise setup

### PR DESCRIPTION
**Pull request by derLalla**
### Motivation and Context

The documentation was missing the creation of a token for the version control system.

### CODE REVIEW

**Review by anditurdiu — APPROVED** [review]
Looks good 👍

**Review by sjagla27 — DISMISSED** [review]
Looks good.

**Review by FuchsDominik — APPROVED** [review]
Looks good to me.

**Review by kloessst — CHANGES_REQUESTED** [review]
Looks good. Just found a small duplication.

File: docs/dev/setup/bamboo-bitbucket-jira.rst [hunk 1]
```diff
@@ -170,24 +170,44 @@ under ``localhost:7990``.
...
+      will be removed in the future.
+
+      - Log in as the admin user and go to Bamboo -> View Profile (top right corner) -> Manage account -> Personal access tokens -> Create token
+
+      - Copy the generated token to your ``application-local.yml``:
+
+      .. code:: yaml
+
+          artemis:
+              continuous-integration:
```
Comment (fde312) [inline docs/dev/setup/bamboo-bitbucket-jira.rst:207]: Should be `version-control:` right?

**Review by fde312 — CHANGES_REQUESTED** [review]

File: docs/dev/setup/bamboo-bitbucket-jira.rst (hunk 1 above)
Comment (derLalla) [reply docs/dev/setup/bamboo-bitbucket-jira.rst]: yep, it's fixed :)

File: docs/dev/setup/bamboo-bitbucket-jira.rst [hunk 2]
```diff
@@ -170,24 +170,44 @@ under ``localhost:7990``.
...
+                  user: <username>
+                  password: <password>
+                  token: #insert the token here
+
+   9.2 Personal access token for Bitbucket.
+   
+      While username and password can still be used as a fallback, this option is already marked as deprecated and
+      will be removed in the future.
```
[the remark this answers is not in the document]
Comment (derLalla) [reply docs/dev/setup/bamboo-bitbucket-jira.rst]: thanks! I've fixed that.

**Review by kloessst — DISMISSED** [review]
Looks good to me!

**Review by fde312 — APPROVED** [review]
LGTM! 👍

File: docs/dev/setup/bamboo-bitbucket-jira.rst [hunk 3]
```diff
@@ -170,24 +170,41 @@ under ``localhost:7990``.
...
+   9.1 Personal access token for Bamboo.
+   
+      - Log in as the admin user and go to Bamboo -> Profile (top right corner) -> Personal access tokens -> Create token
 
-   - Copy the generated token to your ``application-local.yml``:
+          .. figure:: bamboo-bitbucket-jira/bamboo-create-token.png
+             :align: center
 
-   .. code:: yaml
+      - Copy the generated token to your ``application-local.yml``:
```
Comment (krusche) [inline docs/dev/setup/bamboo-bitbucket-jira.rst:185]: The rest of the documentation mentions the file `application-artemis.yml`, we should also do this here!

Comment (krusche) [reply docs/dev/setup/bamboo-bitbucket-jira.rst:185]:
```suggestion
      - Copy the generated token to the file ``application-artemis.yml``:
```

File: docs/dev/setup/bamboo-bitbucket-jira.rst [hunk 4]
```diff
@@ -170,24 +170,41 @@ under ``localhost:7990``.
...
+              continuous-integration:
+                  user: <username>
+                  password: <password>
+                  token: #insert the token here
+
+   9.2 Personal access token for Bitbucket.
+
+      - Log in as the admin user and go to Bitbucket -> View Profile (top right corner) -> Manage account -> Personal access tokens -> Create token
+
+      - Copy the generated token to your ``application-local.yml``:
```
Comment (krusche) [inline docs/dev/setup/bamboo-bitbucket-jira.rst:199]:
```suggestion
      - Insert the generated token into the file ``application-artemis.yml`` in the section ``version-control``:
```

**Review by krusche — APPROVED** [review]

**Pull request merged**